client/src/components/postCard/HoverLikes.tsx

```diff
@@ -0,0 +1,21 @@
+import LikeItem from './LikeItem';
+
+interface Like{
+  id:number
+  User:User
+}
+interface User{
+  id:number
+  name:string
+  avatar:string
+}
+interface props{
+  likes:Array<Like>
+}
+const HoverLikes = ({ likes }:props) => (
```

Comment:
if your props are not to many you can inline their type
```suggestion
const HoverLikes = ({ likes }:{ likes: Array<Like> }) => (
```